deployment/helm-charts/codattle/templates/keycloak-service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: keycloak
+  labels:
+    app: keycloak
+    name: keycloak
+spec:
+  type: NodePort
```

Comment:
Jeżeli chcemy udostępnić jakiś serwis na świat, to nie powinnyśmi używać `NodePort` tylko `Ingress`. Powinieneś dodać odpowiednie przekierowanie w pliku `ingress.yaml`.